node-module-system/calculate.ts

```diff
@@ -0,0 +1,26 @@
+import { addTwoNumbers } from './add.js';
+import { subtractTwoNumbers } from './subtract.js';
+import { multiplyTwoNumbers } from './multiply.js';
+import { divideTwoNumbers } from './divide.js';
+
+if (process.argv[3] === 'plus') {
+  console.log(
+    'result:',
+    addTwoNumbers(Number(process.argv[2]), Number(process.argv[4]))
+  );
+} else if (process.argv[3] === 'minus') {
+  console.log(
+    'result:',
+    subtractTwoNumbers(Number(process.argv[2]), Number(process.argv[4]))
+  );
+} else if (process.argv[3] === 'times') {
+  console.log(
+    'result:',
+    multiplyTwoNumbers(Number(process.argv[2]), Number(process.argv[4]))
+  );
+} else if (process.argv[3] === 'over') {
+  console.log(
+    'result:',
+    divideTwoNumbers(Number(process.argv[2]), Number(process.argv[4]))
+  );
+}
```

Comment:
This solution works. It could be improved, though:
- You have quite a few references to `process.argv[2]`, `process.argv[3]`, and `process.argv[4]`. It would be nice to eliminate all those references. Doing so would also make it easier to enhance the code if you decide to change the way you want arguments passed.
- There is no error handling. For example, what happens if the misspell the operation?
- You also have code duplication in the way you format the logging. If you change your mind on how you do that, you'll need to change the code in several places.